static/scss/base/page.scss

```diff
@@ -3,6 +3,7 @@ body {
 }
 
 .container {
+    overflow: auto;
```

Comment:
This has the side-effect of pushing the Domains of Expertise menu down, creating a vertically scrolling navbar.

Reply:
My markup has gotten very hairy, and this bug is a result. This works now, but I think I'll need to refactor this soon.